python/python-core/utilities-ii/working-with-junk-data.md

```diff
@@ -65,8 +63,7 @@ print(s.find_longest_match(0, 5, 0, 9))
 Complete the `SequenceMatcher`  constructor such that empty spaces are treated as junk:
 
 ```python
-s = SequenceMatcher(??? x: x == ???,
-    “ abcd”, “abcd abcd”)
+s = SequenceMatcher(??? x: x == ???, “ abcd”, “abcd abcd”)
```

Comment:
The quotes here look different?

Reply:
The PR that changes the quotes is [this one](https://github.com/enkidevs/curriculum/pull/2977)